docs/user-guide/configuration/index.md

```diff
@@ -32,28 +28,44 @@ You can access information about properties by hovering over the property name.
         <a href="#steps"><span class="key">steps</span>:
     - <span class="key">init</span>: <span class="value">npm install</span>
     - <span class="key">test</span>: <span class="value">npm test</span></a>
     <a href="#jobs"><span class="key">publish</span>:
+    <span class="key">requires</span>: <span class="value">main</span>
     <span class="key">image</span>: <span class="value">node:6</span>
     <span class="key">steps</span>:
         - <span class="key">publish</span>: <span class="value">npm publish</span></a>
     <a href="#jobs"><span class="key">deploy-west</span>:
+    <span class="key">requires</span>: <span class="value">publish</span>
     <span class="key">image</span>: <span class="value">node:6</span>
     <span class="key">environment</span>:
         <span class="key">DEPLOY_ENV</span>: <span class="value">west</span>
     <span class="key">steps</span>:
         - <span class="key">init</span>: <span class="value">npm install</span>
-        - <span class="key">publish</span>: <span class="value">npm deploy</span></a>
+        - <span class="key">deploy</span>: <span class="value">npm deploy</span></a>
+    <a href="#jobs"><span class="key">deploy-east</span>:
+    <span class="key">requires</span>: <span class="value">publish</span>
+    <span class="key">image</span>: <span class="value">node:6</span>
+    <span class="key">environment</span>:
+        <span class="key">DEPLOY_ENV</span>: <span class="value">east</span>
+    <span class="key">steps</span>:
+        - <span class="key">init</span>: <span class="value">npm install</span>
+        - <span class="key">deploy</span>: <span class="value">npm deploy</span></a>
+    <a href="#jobs"><span class="key">finished</span>:
+    <span class="key">requires</span>: <span class="value">[deploy-west, deploy-east]</span>
+    <span class="key">image</span>: <span class="value">node:6</span>
+    <span class="key">steps</span>:
+        - <span class="key">echo</span>: <span class="value">echo done</span></a>
     <a href="#jobs">...</a>
 </pre>
     <div class="yaml-side">
-        <div id="workflow" class="hidden">
-            <h4>Workflow</h4>
-            <p>Defines the order of jobs that are executed for the project. All jobs referenced by the workflow must be defined in the jobs section.</p>
+        <div id="requires" class="hidden">
+            <h4>Requires</h4>
+            <p>A single job name or array of jobs that will trigger the job to run. Defines the order in which the jobs in the pipeline are run. Jobs defined with "requires: ~pr" are started by pull-request events. Jobs defined with "requires: ~commit" are started by push events. Jobs defined with "requires: ~sd@123:main" are started by job "main" from pipeline "123". Jobs defined with "requires: [deploy-west, deploy-east] are started after "deploy-west" and "deploy-east" are both done running successfully. "Note: ~ jobs denote an OR functionality, jobs without a ~ denote join functionality.</p>
```

Comment:
I think we can leave this out "Defines the order in which the jobs in the pipeline are run."  It might be confusing.